### DISCUSSION TITLE
Cloud limits

### DISCUSSION THREAD

**Discussion by Bazz-tube** [forkphorus/cloud-server#130] [Q&A]
### Hello 👋

I was wondering if it was possible to edit/remove the cloud variable limits on servers,  also a JSON with all the limits like length, ussage of letters and special characters etc. would be nice. I understand if this isn't possible or is a bad idea.
Thank you for reading this and I hope you have a wonderful rest of the day, once again thank you and bye 👋

**Comment by memer3638** [comment]
something that would work is every log is using up your memory so it really depends on that, i will try to look into it and update whenever i can as soon as possible.

**Comment by memer3638** [comment]
As far as what i've seen it just stores on memory but if you go to "cloud-server/src/server.js" inside the files you can edit it on line 15 by editing the numbers that show it like this for example "1024*1024" and change it to something like "2048*2048" and it would double

**Comment by squirticulous** [comment]
if you're talking about permanently storing cloud data, that was being worked on under the persistency branch which has pull request associated with it https://github.com/TurboWarp/cloud-server/pull/107 but according to the pull request garbo muffin isn't working on it right now, though im sure you could just clone the persitency branch and run it yourself
